trackers/core/deepsort/feature_extractor.py

```diff
@@ -1,169 +0,0 @@
-from typing import Optional, Tuple, Union
-
-import numpy as np
-import supervision as sv
-import timm
-import torch
-import torch.nn as nn
-import torchvision.transforms as transforms
-
-from trackers.utils.torch_utils import parse_device_spec
-
-
-class FeatureExtractionBackbone(nn.Module):
```

Comment:
@soumik12345 we need to make sure that all content and examples using `DeepSORTFeatureExtractor` are migrated to `ReIDModel`.